workflow-dockstore-metrics/workflow_dockstore_metrics/pyvenv.cfg

```diff
@@ -0,0 +1,3 @@
+home = /Users/mcnatt/.pyenv/versions/3.9.16/bin
```

Comment:
I do not think you need to include this file, it also has a conflicting python version compared to the poetry one

Reply:
I think you can still remove this file